modules/core/src/main/scala/org/scalasteward/core/util/HttpJsonClient.scala

```diff
@@ -38,5 +40,33 @@ final class HttpJsonClient[F[_]: Sync](
     post[A](uri, modify.compose(_.withEntity(body)(jsonEncoderOf[F, B])))
 
   private def request[A: Decoder](method: Method, uri: Uri, modify: ModReq): F[A] =
-    client.expect[A](modify(Request[F](method, uri)))(jsonOf[F, A])
+    client.expectOr[A](modify(Request[F](method, uri)))(
+      resp => ToUnexpectedResponse(uri, method, resp)
+    )(jsonOf[F, A])
+
+  private def ToUnexpectedResponse(uri: Uri, method: Method, response: Response[F])(
+      implicit F: MonadThrowable[F]
```

Comment:
How should I summon an F?  `implicitly` is not used at all in the project, the only thing I saw so far is using an implicit parameter

Reply:
You can just call `map` on `fBody`: `fBody.map(body => ...)`. This works because of the `cats.implicits._` import which provides syntax for `map` and the `Sync` (`Sync` is a subclass of `Functor`) instance that is already in scope.

Reply:
ah I see, the `import cats.implicits._` was the missing key, I improved the PR